global-configuration/src/main/resources/archetype-resources/src/test/java/SampleConfigurationTest.java

```diff
@@ -0,0 +1,30 @@
+package $package;
+
+import com.gargoylesoftware.htmlunit.html.HtmlForm;
+import com.gargoylesoftware.htmlunit.html.HtmlTextInput;
+import org.junit.Test;
+import static org.junit.Assert.*;
+import org.junit.Rule;
+import org.jvnet.hudson.test.RestartableJenkinsRule;
+
+public class SampleConfigurationTest {
+
+    @Rule
+    public RestartableJenkinsRule rr = new RestartableJenkinsRule();
+
+    @Test
+    public void uiAndStorage() {
```

Comment:
Exercises all the mistakes I could think to make: missing `load`; missing `save`; misnamed or absent getter/setter; misnamed `textbox`.

Reply:
Possibly worth adding these thoughts as a comment so that people who are new to this can actually think about what common/easy mistakes they may make.

Reply:
Very good point, will do.